presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -175,6 +180,16 @@ private HiveUtil()
                 .filter(name -> name.startsWith("serialization."))
                 .forEach(name -> jobConf.set(name, schema.getProperty(name)));
 
+        // add Airlift LZO and LZOP to head of codecs list as to not override existing entries
+        List<String> codecs = new ArrayList<>(Splitter.on(",").trimResults().omitEmptyStrings().splitToList(jobConf.get("io.compression.codecs", "")));
```

Comment:
You could do
```java
List<String> codes = newArrayList(Splitter.on(",").trimResults().omitEmptyStrings().split(jobConf.get("io.compression.codecs", "")));
```
To avoid building the list twice